test/test.js

```diff
@@ -182,7 +182,7 @@ describe('LiteAPI SDK Test Suite', function() {
 
   it('should create a new voucher', async function() {
     const data = {
-      voucher_code: 'em4z9z7l',
+      voucher_code: 'be49z8l',
```

Comment:
Wouldn't this fail when called multiple times?

Reply:
That's why I told you the ID of the voucher should be random, so you can run the test multiple times and they will pass.